seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumComJSRunner.scala

```diff
@@ -37,12 +41,18 @@ class SeleniumComJSRunner(browserProvider: SeleniumBrowser,
   }
 
   def send(msg: String): Unit = {
+    if (comClosed)
+      throw new ComJSEnv.ComClosedException
     awaitForBrowser()
-    browser.getWebDriver.executeAsyncScript(sendScript, msg)
+    val encodedMsg =
+      msg.replace("\u0007", "\u0007\u0007").replace('\u0000', '\u0007')
```

Comment:
What if `msg.contains("\u0000\u0000")`?

I suggest: `msg.replace("\u0007", "\u0007\u0007").replace("\u0000", "\u00070")`.

Btw, why `\u0007`? Why not something a bit more readable and conventional like `\\`: `msg.replace("\\", "\\\\").replace("\u0000", "\\0")`?